content/en/docs/components/serving/bentoml.md

```diff
@@ -0,0 +1,258 @@
++++
+title = "BentoML"
+description = "Model serving with BentoML"
+weight = 40
++++
+
+{{% stable-status %}}
+
+## Model serving with BentoML
```

Comment:
Recommendation: Remove this heading, as it's basically a repeat of the page title/description. Then change all the other headings from level 3 to level 2.